src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx

```diff
@@ -98,7 +102,8 @@ const ClusterDashboard = () => {
   };
 
   const moveToFeeRecipient = () => {
-    navigate(config.routes.SSV.MY_ACCOUNT.CLUSTER.FEE_RECIPIENT);
+    accountStore.getFeeRecipientAddress(walletStore.accountAddress)
+        .finally(() => navigate(config.routes.SSV.MY_ACCOUNT.CLUSTER.FEE_RECIPIENT));
```

Comment:
Do we have some indication here?
If request will take some time, half second, second or two seconds, user will want to go fee recipient and will have some strange delay. Maybe we can put spinner inside of the button and disable button for that time when fee recipient is loading

Reply:
done